packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -27,31 +23,22 @@ const GuidanceWrapper = styled.div`
 `;
 
 const Content = styled.div`
-  display: flex;
-  align-items: center;
-  padding: ${GEL_SPACING_DBL};
+  padding: ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_DBL};
+  }
 `;
 
 const GuidanceMessage = styled.strong`
-  padding-left: ${GEL_SPACING_HLF};
   font-weight: normal;
 `;
 
-const IconWrapper = styled.div`
-  > svg {
-    color: ${C_WHITE};
-    fill: currentColor;
-    height: ${GEL_SPACING_TRPL};
-    width: ${GEL_SPACING_TRPL};
-  }
-`;
-
 const Guidance = ({ guidanceMessage, service, children }) => {
   return (
     <StyledGuidance service={service}>
       <GuidanceWrapper>
```

Comment:
On a similar theme, these two "wrappers" could be combined. You could:
- Prune `StyledGuidance` and
- Move the `service={server}` onto `GuidanceWrapper`.